support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -94,7 +94,12 @@ class CustomActionBuilders(
       } else {
         result
       }
-    }
+    }.recoverWith({ case throwable: Throwable =>
+      SafeLogger.error(scrub"pushing alarm metric - 5xx response caused by ${throwable}")
+      val cloudwatchEvent = AwsCloudWatchMetricSetup.serverSideCreateFailure(stage)
+      AwsCloudWatchMetricPut(AwsCloudWatchMetricPut.client)(cloudwatchEvent)
+      Future.failed(throwable)
```

Comment:
this makes it "recover" back to the original failure, as we're returning what was passed in.